sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/PageRangeItem.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.models;
+
+import com.azure.core.http.HttpRange;
+import com.azure.storage.blob.specialized.PageBlobClient;
+
+/**
+ * Represents a page range on a page blob returned by {@link PageBlobClient#listPageRanges(BlobRange)}.
+ */
+public final class PageRangeItem {
+    private final HttpRange range;
```

Comment:
Did we update HttpRange to support inclusive ranges?

Reply:
I have not made that change. Did you want me to submit that update?

Reply:
Would this be an implicit or explicit change to `HttpRange`? Do we know of anything relying in the end range being exclusive?

Reply:
We'll need to extend HttpRange to have an indicator to specify whether the range is inclusive or exclusive. By default, it would be exclusive. We'll probably have `HttpRange(long offset, long length, boolean isRangeInclusive)`

Reply:
This is good with me, it'll help `HttpRange` cover additional scenarios

Reply:
Discussed offline that we'll merge this as is for now with a numerical adjustment and then submit another PR once core makes the update.